### PR TITLE
Fix game board printing problem

### PR DESCRIPTION
**Pull request by KrzysiekJodlowski**
In the App there was two issues with the custom board size presentation:
- BoardPrinter was not reinitialized so the upper side length was set
always to 3
- while loop condition was checking wrong type of input, which was
resulting with NoSuchElementException

### CODE REVIEW

**Review by Haremun — APPROVED** [review]
Just fast bugfix.

File: src/main/java/com/github/krzysiekjodlowski/ox/App.java [hunk 1]
```diff
@@ -38,9 +38,10 @@ public static void main(final String[] args)
             Scanner scanner = new Scanner(
                     new FileInputStream(new File(args[0]))
             );
-            while (scanner.hasNextLine()) {
```
Comment (michal-sydoryk) [inline src/main/java/com/github/krzysiekjodlowski/ox/App.java:41]: great change!

**Review by michal-sydoryk — APPROVED** [review]
nice

**Comment by KrzysiekJodlowski** [comment]
Thank you:P

**Pull request merged**